public/modules/custom/grants_metadata/src/Controller/ApplicationStatusCheckController.php

```diff
@@ -82,7 +84,7 @@ public static function create(ContainerInterface $container) {
   /**
    * Builds the response.
    *
-   * @return Symfony\Component\HttpFoundation\JsonResponse
+   * @return JsonResponse
```

Comment:
Tällä hetkellä taitaa koodissa melko yhdenmukaisesti olla nämä muuttujien tyypit dokumentoitu täyspitkässä muodossa. Olisikohan syytä pitäytyä siinä ja laittaa vaan kautta-viiva?

```suggestion
   * @return \Symfony\Component\HttpFoundation\JsonResponse
```